.kitchen.yml

```diff
@@ -1,6 +1,6 @@
 ---
 <% vagrant = system('gem list -i kitchen-vagrant 2>/dev/null >/dev/null') %>
-<% version = '2017.7.6' %>
+<% version = '2018.3.3' %>
```

Comment:
We need to make sure the salt-jenkins is merged forward to master for this.

Reply:
@gtmanfred Ok can I get a review here?

https://github.com/saltstack/salt-jenkins/pull/1187

Then I can merge-forward to `master` in salt-jenkins.